wolfcrypt/src/aes.c

```diff
@@ -356,7 +356,7 @@
 
         HAL_CRYP_DeInit(&hcryp);
     #else
-        #error AES Decrypt not implemented for STM32 StdPeri lib
+        #warning AES Decrypt not implemented for STM32 StdPeri lib
```

Comment:
I'll update this back to error. Also the #warning syntax doesn't work with all compilers.